src/alpine/3.19/amd64/Dockerfile

```diff
@@ -37,6 +46,20 @@ RUN apk update && apk add --no-cache \
         shadow \
         sudo \
         tzdata \
+        userspace-rcu \
         util-linux-dev \
         which \
         zlib-dev
+
+RUN apk -X https://dl-cdn.alpinelinux.org/alpine/edge/main add --no-cache \
+    lttng-ust
+
+RUN curl -L https://github.com/PowerShell/PowerShell/releases/download/v7.4.2/powershell-7.4.2-linux-musl-x64.tar.gz -o /tmp/powershell.tar.gz
```

Comment:
I'm not a fan of pinning to a specific version. This will very quickly become out of date and trigger vulnerability reports. In fact, 7.4.3 was just released yesterday. So this is already out of date.

Ideally, there would exist a PowerShell image for Alpine 3.19 that could be used as the base instead of `alpine:3.19` but no such image exists.

A better solution would be to query the [GitHub releases API](https://[api.github.com/repos/powershell/powershell/releases) to find the latest 7.4.x version and find the asset to download from there.

Reply:
If we didn't want a very specific major.minor, you could also use: `https://github.com/PowerShell/PowerShell/releases/latest` which redirects to the latest tag

Reply:
I've changed this to use the latest tag. There's no way to query a specific line via the releases (unless you want to go into very jquery land). instead i'm querying the latest, then verifying it's 7.x. It will be a stable release, not a preview release.